packages-exp/auth-exp/src/api/index.ts

```diff
@@ -182,6 +187,16 @@ export async function _performSignInRequest<T, V extends IdTokenResponse>(
   return serverResponse;
 }
 
+export function _getFinalTarget(auth: AuthCore, url: string): string {
+  const { emulator } = auth.config;
+  if (!emulator) {
+    return url;
+  }
+
+  const urlWithoutScheme = url.replace(/^.*:\/\//, '');
```

Comment:
instead of doing string regex, can you just pass all the parts to this function and have it combine everything once figuring out the correct host?

Reply:
Done